kayenta-web/src/main/java/com/netflix/kayenta/Main.java

```diff
@@ -56,7 +58,8 @@
   S3Configuration.class,
   StackdriverConfiguration.class,
   WebConfiguration.class,
-  NetflixJudgeConfiguration.class
+  NetflixJudgeConfiguration.class,
+  InfluxdbConfiguration.class
```

Comment:
Sort-order is off here (please move the `NetflixJudgeConfiguration` entry to its proper location as well).